libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -173,9 +182,15 @@ class EmbeddingStore(DynamicBase):
             """This is the base model for SQL vector store."""
 
             __tablename__ = name
-            __table_args__ = {"schema": schema}
+            __table_args__ = (
+                PrimaryKeyConstraint("id", mssql_clustered=False),
```

Comment:
if the primary key is also NON-clustered, there is no clustered index at all? What would the clustered index be if were to have it?

Reply:
No, we do not have any clustered index.

The primary key makes sense as non-clustered because they are random values.

If we were to have a clustered index, this would have been best suited for the custom_id column. However, there doesn't seem to be a need for this since the ultimate goal of this table is to do a semantic search on it. We don't need the entries to be sorted in any order.

Reply:
can you please mention this as a comment in the PR description/in code why clustered index is not added

Reply:
Sure